firmware/common/libs/fault_check/oscc_check.h

```diff
@@ -10,19 +10,60 @@
 
 #include <stdint.h>
 
+typedef struct {
+    bool condition_active;              //!< Fault condition active flag
```

Comment:
Maybe call this something like monitoring_active to denote difference between this bool and the bool in the function calls also called condition_active?

Reply:
Love it, will do